.github/workflows/ui_tests.yml

```diff
@@ -52,6 +54,14 @@ jobs:
     timeout-minutes: 120
 
     steps:
+    - name: Install screenresolution
+      if: matrix.runner == 'macos-15-xlarge'
+      run: brew install screenresolution
+
+    - name: Change screen resolution
+      if: matrix.runner == 'macos-15-xlarge'
+      run: screenresolution set 1920x1080x32@60
```

Comment:
We could perhaps do this for all runners 🤔 My worry here is that it will bite us back when macos-16 is introduced and we'd have to come back here and update the if statement, so maybe it makes sense to just do this unconditionally to spare future headaches :)